cve_bin_tool/cli.py

```diff
@@ -19,10 +19,11 @@
 
 import pkg_resources
 
-from .scanner import Scanner
+from .csv_scanner import CVEScanner
```

Comment:
I think it should be cve_scanner if I'm not wrong

Reply:
Yep typo!